input/fsh/profiles-resources/CHCoreImmunization.fsh

```diff
@@ -0,0 +1,50 @@
+Profile: CHCoreImmunization
+Parent: Immunization
+Id: ch-core-immunization
+Title: "CH Core Immunization"
+Description: "CH Core definition of the immunization."
+* ^version = "0.2.0"
+* ^status = #active
+* ^experimental = false
+* . ^short = "CH Core Immunization"
+* extension ^slicing.discriminator.type = #value
+* extension ^slicing.discriminator.path = "url"
+* extension ^slicing.rules = #open
+* extension contains
+    Author named recorder 0..1 and
+    EntryResourceCrossReferences named relatesTo 0..1 
+* extension[recorder] ^definition = "The recorder Reference of the immunization. May be a Practitioner or a Patient"
+* extension[relatesTo] ^definition = "The definition of a relation of this immunization entry in this document to an immunization entry in an other document."
```

Comment:
relatesTo: nur im Dokumentkontext wichtig also CH VACD oder auch sonst?

Reply:
hatte in ins core genommen, weil extension auch im core, doch ja ist documenten context abhängig.